app/javascript/spec/dual-list-select/helpers.spec.js

```diff
@@ -0,0 +1,73 @@
+import {
+  isEmpty,
+  leftValues,
+  leftSubmittedKeys,
+  addedLeftKeys,
+  addedRightKeys,
+} from '../../components/dual-list-select/helpers';
+
+describe('Dual list select helpers', () => {
+  describe('isEmpty', () => {
+    it('should return true if object is empty ', () => {
+      expect(isEmpty({})).toBeTruthy();
```

Comment:
If you know what value is returned its better to compare it directly. Truthy and falsy should be used for cases when it isn't clear what method can return.